docs/index.rst

```diff
@@ -72,18 +72,31 @@ The best way to get started with Parcels is to have a look at the Jupyter notebo
 Installing Parcels
 ===================================
 
-The latest version of Parcels, including tests and examples, 
-can be obtained directly from github via::
-    git clone https://github.com/OceanParcels/parcels.git
-    cd parcels; pip install -r requirements.txt
-    python scripts/pull_data.py
-    export PYTHONPATH="$PYTHONPATH:$PWD"
-
-In order for Parcels to work from any directory, add the following line to 
-your ~/.bash_profile::
-    export PYTHONPATH="$PYTHONPATH:$PWD"
-
-Note that a functional NetCDF install is required.
+Parcels depends on a working Python installation, a netCDF installation, a C
+compiler, and various Python packages.  These requirements can be met in
+different ways:
+
+1. Use Anaconda and Conda-Forge for all requirements.  This is the
+   “batteries-included” solution probably suitable for most users.
+
+2. Use an existing netCDF installation and an existing C compiler and just get
+   the Python part of the dependencies.  This may be a suitable solution for
+   more experienced users who want to control the details.
+
+Installing Parcels and all dependencies with Anaconda and Conda-Forge
+-----------------------------------
+
+Install Anaconda's Miniconda following the steps at https://conda.io/docs/user-guide/install/
+
+Download the `environment.yml <https://raw.githubusercontent.com/OceanParcels/parcels/install_via_conda/environment.yml>`_ file and place it in the directory where you want to install Parcels. Then, run::
```

Comment:
I think the "place it in the directory where you want to install Parcels" is misleading as the `environment.yml` only has to be present while the environment is created. Would it be still understandable, if the whole thing would be truncated to
> Download the ... file and run::

Another thing to keep in mind: The link should point to
<https://raw.githubusercontent.com/OceanParcels/parcels/master/environment.yml>
in the end.

Reply:
Good point. I've now also added a check of broken files in the documentation using the `sphinx linkcheck` functionality. So even if we forget, travis will complain